scripts/build_utils.sh

```diff
@@ -1004,10 +1004,11 @@ build_debs() {
             $venv/chacractl binary ${chacra_flags} create ${chacra_endpoint}
 
         # extract cephadm if it exists
-        if [ -f release/${vers}/cephadm_${vers}*.deb ] ; then
-            dpkg-deb --fsys-tarfile release/${vers}/cephadm_${vers}*.deb | tar -x -f - --strip-components=3 ./usr/sbin/cephadm
+        for file in release/${vers}/cephadm_${vers}*.deb; do
```

Comment:
Is it logically OK for this loop to execute for >1 matching file, even if that's unlikely?

Reply:
Yeah, isn't that just going to extract cephadm from the last one if there are >1?

Reply:
I was unsure, so I had inserted a `break` below

Reply:
I missed that, thanks.